### ISSUE TITLE
ERROR: Could not find a version that satisfies the requirement tbb==2021.13.0 

### ISSUE THREAD

**Issue by abdullahmoosa** [uxlfoundation/oneTBB#1505]
# Summary
I am using python 3.12.4 in MacOS 14.6.1. when installing tbb version 2021.13.0 I am facing this issue : ERROR: Could not find a version that satisfies the requirement tbb==2021.13.0 (from versions: 2018.0.4.1, 2019.0, 2021.1.1, 2021.2.0, 2021.3.0, 2021.4.0, 2021.5.0, 2021.6.0, 2021.7.0, 2021.7.1, 2021.8.0, 2021.9.0, 2021.10.0)
ERROR: No matching distribution found for tbb==2021.13.0
# Version
onetbb version : 2021.13.0,

# Environment
Device : MacOS 14.6.1.
python version : 3.12.4
pip version : 24.2

# Observed Behavior
ERROR: Could not find a version that satisfies the requirement tbb==2021.13.0 (from versions: 2018.0.4.1, 2019.0, 2021.1.1, 2021.2.0, 2021.3.0, 2021.4.0, 2021.5.0, 2021.6.0, 2021.7.0, 2021.7.1, 2021.8.0, 2021.9.0, 2021.10.0)
ERROR: No matching distribution found for tbb==2021.13.0
# Expected Behavior
Install tbb properly
# Steps To Reproduce
Ensure you are in MacOS 14.6.1,
ensure python version 3.12.4.

create a virtual environment,
enable it.
then pip install tbb==2021.13.0

**Comment by omalyshe** [comment]
@abdullahmoosa That is correct behavior because the latest oneTBB version available in pip is 2021.10.0 - https://pypi.org/project/tbb/#history.

**Comment by abdullahmoosa** [comment]
> @abdullahmoosa That is correct behavior because the latest oneTBB version available in pip is 2021.10.0 - https://pypi.org/project/tbb/#history.
> 
> You can download 2021.13.0 for macOS from GitHub - https://github.com/oneapi-src/oneTBB/releases/tag/v2021.13.0


Thank you.
But what about [https://pypi.org/project/tbb/2021.13.1/](url) ?

**Comment by omalyshe** [comment]
> > @abdullahmoosa That is correct behavior because the latest oneTBB version available in pip is 2021.10.0 - https://pypi.org/project/tbb/#history.
> > You can download 2021.13.0 for macOS from GitHub - https://github.com/oneapi-src/oneTBB/releases/tag/v2021.13.0
> 
> Thank you. But what about [https://pypi.org/project/tbb/2021.13.1/](url) ?

oneTBB bits in 2021.13.1 are same as in 2021.13.0. So, you can use the package from GitHub as I mentioned.

**Comment by omalyshe** [comment]
As explained in comments - it is not the bug.
